test/test_format.py

```diff
@@ -582,7 +582,7 @@ def test_nice_duration(self):
                          "thirteen hours fifty three minutes twenty seconds")
         self.assertEqual(nice_duration(50000, speech=False), "13:53:20")
         self.assertEqual(nice_duration(500000),
-                         "five days  eighteen hours fifty three minutes twenty seconds")  # nopep8
+                         "five days eighteen hours fifty three minutes twenty seconds")  # nopep8
```

Comment:
On the one hand, `#nopep8` makes me think this test was meant to verify that `nice_duration()` works with odd spaces in the input string, so this shouldn't be fixed.

On the other hand, the fact that the tokenizers work is proof positive that whitespace doesn't matter here, so `#nopep8` could  come out.

Either way, not really relevant to this PR, but it's the kind of thing that linters often catch automatically so you merge it anyway.

Reply:
I'd agree on this one. As it's the only test in the file with a `#nopep8` flag I think it's intentionally there to test odd spacing.
@Tony763 can you switch this back please?